gems/ibm_cloud_global_tagging/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Questions
+If you are having problems using the APIs or have a question about IBM Cloud services,
+please ask a question at
+[Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-cloud).
+
+# Issues
+If you encounter an issue with the project, you are welcome to submit a
+[bug report](https://github.com/IBM-Cloud/global_tagging-ruby-sdk/issues).
```

Comment:
I'm guessing 'IBM-Cloud/global_tagging-ruby-sdk' will be the future home?

Reply:
:+1: good catch, fixed